### ISSUE TITLE
Explaining message instead of empty result table

### ISSUE THREAD

**Issue by MichaelRoeder** [hobbit-project/platform#5]
Until now, an empty result table is shown if
a) the experiment is unknown
b) there are no results for the experiment because it is still running or hasn't been started until now

In both cases, the UI does not get any data for the requested experiment, i.e., the UI can not distinguish between these two cases at the moment. Its current behaviour is to show an empty table.

However, this behaviour is not easy to understand for a user. The UI should add an explanation that either the experiment does not exist or the experiment has not been finished.

**Comment by romankorf** [comment]
We at the GUI side cannot decide if there is some benchmark or not. We however have a field error in the expriment results (ExperimentBean). Our suggestion:

Someone who can identify if there is a job with the given ID is writhing an error message:

Error: Job with ID xyz still pending

or if there is no such job:

Error: No job with the ID xyz

**Comment by MichaelRoeder** [comment]
I totally agree that the GUI can not decide that.

I see two solutions here:
1. Print a message like "The experiment is still pending or does not exist" or "There are no results for this experiment until now."
2. Distinguish between both cases, i.e.,
    1. Request experiment results
    2. If there are no results request the current experiment status
    3. If the experiment can be found in the status message, print "Experiment is still pending" (and put benchmark and system names/URIs). Otherwise, print "Experiment does not exist".

The second solution might look slightly better. However, it creates much more effort (in development as well as at runtime) and is more error-prone (in some cases the GUI would state that an experiment does not exist while it is existing). Taking into account that the first solution would already solve the problem, I would prefer the first one. 
@romankorf What do you think?

**Comment by romankorf** [comment]
Solution provided in bugfix for issue #5

**Comment by MichaelRoeder** [comment]
Reduced the priority since a workaround is available.

A final solution should include a detailed information whether the experiment is in the queue, currently executed or does not exist at all.

**Comment by MichaelRoeder** [comment]
Removed the `bug` tag since there is a workaround.

**Comment by denkv** [comment]
Possible low-handing fruit: web client may reuse the _running experiment_ and _experiment queue_ data from "experiment status" page to display if currently viewed experiment is in queue or being run right now. Maybe even reuse the same information-displaying interface component as in "experiment status" page.

**Comment by MichaelRoeder** [comment]
No, as denkv described above, we would like to "completely" solve this issue, i.e., displaying the real status of an experiment (completed, running, in queue or not-existing) instead of distinguishing only between the two states - experiment is in database or not.
